### ISSUE TITLE
Archive?

### ISSUE THREAD

**Issue by jzf2101** [binder-examples/dockerfile-rstudio#5]
@betatim Suggested this isn't required anymore since we now automatically allow people to use R on Binder?

**Comment by choldgraf** [comment]
I agree it's not necessary anymore, though think it could be useful if we rebranded it as something like "extending Binder for new interfaces" or something like that. Could be inspiration / template for adopting new interfaces in the future. What do you guys think?

**Comment by cboettig** [comment]
@choldgraf good suggestion!  I think it's a nice example for extending with the 'bring your own dockerfile' approach.  I still find often find it easier to just deploy a dockerfile on binder that I already know "works" (i.e. has the correct libraries etc -- I know it's often 'good enough' but I've never liked `apt.txt`-style dependency management without having any knowledge/control over what repos are in `/etc/apt/sources.list`, that can be a fast-track to dependency hell).

**Comment by betatim** [comment]
Reimagining this repo as a "how to add a new UI via custom Dockerfile" sounds like a good plan to me.

What would we need?
* [x] warning that there is a new way of using RStudio so please don't use this for that purpose
* [ ] more description/explanation about the changes/customisations that were made?
* [ ] a new name

**Comment by choldgraf** [comment]
+1 to that. Maybe "new uis with dockerfiles"?
